### ISSUE TITLE
printTypeDefinitions throws TypeError: flatMap is not a function

### ISSUE THREAD

**Issue by MarcCoet** [gatsbyjs/gatsby#19191]
**_Edit: I found the bug. See second comment._**

## Description

I am trying to use `gatsby-plugin-schema-snapshot` and it throws `TypeError: flatMap is not a function`.
This throws as well:
```
exports.createSchemaCustomization = ({ actions }) => {
  actions.printTypeDefinitions({});
};
```

### Steps to reproduce

1. Create a new project.
2. Add `gatsby-plugin-schema-snapshot` to the project or the above mentioned snippet in `gatsby-node.js`
3. `gatsby develop` or `gatsby build`

[Demo repo](https://github.com/MarcCoet/gatsby-flatmap-issue)
[Full log on Netlify](https://app.netlify.com/sites/gatsby-flatmap-issue/deploys/5dbb22ff5aea3100070a4bd9)

I was initially on NodeJS 10 (like Netlify is on the build above). I tried upgrading to Node 13 in case the polyfill was causing the problem but no luck.

### Expected result

Should produce a `schema.gql` file

### Actual result

`TypeError: flatMap is not a function`
See [full log on Netlify](https://app.netlify.com/sites/gatsby-flatmap-issue/deploys/5dbb22ff5aea3100070a4bd9)

### Environment

```
  System:
    OS: Linux 4.19 Debian GNU/Linux 9 (stretch) 9 (stretch)
    CPU: (8) x64 06/8e
    Shell: 5.3.1 - /usr/bin/zsh
  Binaries:
    Node: 13.0.1 - /usr/bin/node
    Yarn: 1.19.1 - /usr/bin/yarn
    npm: 6.12.0 - /usr/bin/npm
  Languages:
    Python: 2.7.13 - /usr/bin/python
  Browsers:
    Firefox: 60.9.0
  npmPackages:
    gatsby: ^2.17.6 => 2.17.6
    gatsby-image: ^2.2.30 => 2.2.30
    gatsby-plugin-manifest: ^2.2.25 => 2.2.25
    gatsby-plugin-offline: ^3.0.17 => 3.0.17
    gatsby-plugin-react-helmet: ^3.1.13 => 3.1.13
    gatsby-plugin-schema-snapshot: ^1.0.0 => 1.0.0
    gatsby-plugin-sharp: ^2.2.34 => 2.2.34
    gatsby-source-filesystem: ^2.1.35 => 2.1.35
    gatsby-transformer-sharp: ^2.3.1 => 2.3.1
```

**Comment by MarcCoet** [comment]
What is strange is that [I tried the snippet I mention a couple months back](https://github.com/gatsbyjs/gatsby/issues/3344#issuecomment-531540934) and all was working fine. I tried downgrading the `gatsby` package to the version I used back then but it didn't work either. Maybe a dependency changed...?

**Comment by MarcCoet** [comment]
Oh I've found the bug:
[flatMap is indeed a function](https://github.com/graphql/graphql-js/blob/master/src/polyfills/flatMap.js#L13)
It is a default export.
In gatsby, it is [not imported as default](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/schema/print.js#L188)

FIX:
```javascript
- const { flatMap } = require(`graphql/polyfills/flatMap`)

+ const flatMap = require(`graphql/polyfills/flatMap`).default
```

I don't have time for a PR right now, sorry... =/

**Comment by MarcCoet** [comment]
Such a simple PR... (#19208) I tried creating it from Github UI directly.
Please tell me if it is ok. I know I am supposed to fork and so on but this makes the process so much easier.

**Comment by stefanprobst** [comment]
should be fixed by #19208
